packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -79,6 +81,13 @@ export const addAuction = async (
     auctioneerInstallationP,
   ]);
 
+  // save the auctioneer instance so we can manage it later
+  const retiredContractInstances = await retiredContractInstancesP;
+  const legacyInstance = legacyKit.instance;
+
+  const boardID = await E(board).getId(legacyInstance);
+  retiredContractInstances.init(`auction-${boardID}`, legacyInstance);
```

Comment:
`legacyInstance` isn't much shorter than `legacyKit.instance`. And `retiredContractInstances` is only used once. consider,
```suggestion
  const boardID = await E(board).getId(legacyKit.instance);
  await E(retiredContractInstancesP).init(`auction-${boardID}`, legacyKit.instance);
```

Reply:
done.